sdk/core/test/integration/NetworkMember.test.ts

```diff
@@ -1094,4 +1094,49 @@ describe('CommonNetworkMember', () => {
     const commonNetworkMember = new AffinidiWallet(password, encryptedSeedElem, options)
     await commonNetworkMember.deleteAllCredentials()
   })
+
+  it('#createWallet should return wallet with resolvable did (elem)', async () => {
```

Comment:
I'd also suggest having yet another test: `createWallet`, then `openWalletByEncryptedSeed`, and make sure the two return the same did.